src/components/qrcode-scanner/QRCodeScanner.js

```diff
@@ -1,72 +1,81 @@
-import React, {
-  Fragment,
-  useCallback,
-  useEffect,
-  useRef,
-  useState,
-} from 'react';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
 import { RNCamera } from 'react-native-camera';
 import { useIsEmulator } from 'react-native-device-info';
+import * as Permissions from 'react-native-permissions';
```

Comment:
we're trying not to use this kind of imports

Reply:
The naming of functions in the Permissions module is really bad. I'll use "as" as from now